homeassistant/components/minecraft_server/config_flow.py

```diff
@@ -55,38 +56,46 @@ class MinecraftServerConfigFlow(ConfigFlow, domain=DOMAIN):
                     partial(getmac.get_mac_address, **params)
```

Comment:
I am a bit surprised that mac address is required. It's totally fine not to have a unique ID.

Reply:
You mean in case of an IP address only I don't set the unique ID? But then the user can't rename the entities, right? And how would I then prevent configuration of the same host? Currently I use the unique ID for that. Also I would like to reach 🥇 as the integration quality scale in the future...

Reply:
I think that we could use SRV records as a unique ID. Surprised the server doesn't have one :(